pages/services/hdfs/2.5.0-2.6.0-cdh5.16.3/release-notes/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+excerpt: Release notes for v. 2.5.0-2.6.0-cdh5.16.3
+title: Release Notes
+menuWeight: 10
+model: /services/hdfs/data.yml
+render: mustache
+---
+## Version 2.5.0-2.6.0-cdh5.16.3
+
+### Bug Fixes
+- Fix for JournalNodes failing to establish quorum when paxos folder is missing.
+- Hadoop `ipc.maximum.data.length` property is now available in service configuration.
+
+### Improvements
+- A validator has been added to check the package has a unique name and follows package naming guidelines on universe.
```

Comment:
Suggestion: `- Added validation to verify package names are unique and follow Universe guidelines`

Reply:
Thanks, fixed.